rc_dev

```diff
@@ -17,6 +17,7 @@ export PUBLIC_BUCKET_HOST=public.dev.bgdi.ch
 export ROBOTS_FILE=robots.txt
 export SERVER_PORT=6543
 export SPHINXHOST=service-sphinxsearch.dev.bgdi.ch
+export SPHINXHOST=10.220.4.170
```

Comment:
please do not forget to let C2C create a new instance for dev with correct puppet resources and test that the install is perfectly functional before creating the new int and prod machines.